projects/developer/src/app/common/components/temporal-filter/component.ts

```diff
@@ -1,57 +1,35 @@
-import { Component, EventEmitter, Input, OnInit, OnDestroy, Output } from '@angular/core';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { MessageService } from 'primeng/components/common/messageservice';
-import { Subscription, Observable } from 'rxjs';
+import { MenuItem } from 'primeng/api';
 import 'rxjs/add/observable/timer';
```

Comment:
It looked to me like most of the timer code was moved into the live-range-selector component. Can this import be pruned?